internal/controller/knative.go

```diff
@@ -38,70 +39,207 @@ const (
 	KnativeServingCRDName         = "knativeservings.operator.knative.dev"
 	KnativeSubscriptionName       = "serverless-operator"
 	KnativeSubscriptionNamespace  = "openshift-serverless"
+	KnativeSubscriptionChannel    = "stable"
 )
```

Comment:
ar those used anywhere else than in the file? If not, remove the `Knative` prefix and make it private

Reply:
Made it private but still preferred to keep the prefix mostly for readability. Also did the same with `sonataflow` consts.